### PR TITLE
Switch base image from Passenger to Puma

### PR DESCRIPTION
**Pull request by phuongdh**

### CODE REVIEW

File: nginx/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,19 @@
 FROM        phusion/baseimage:latest
```
Comment (cjcolvar) [inline nginx/Dockerfile:1]: Is `phusion/baseimage:latest` still the best base image for the nginx container?

File: docker-compose.yml [hunk 2]
```diff
@@ -110,11 +111,15 @@ services:
       - Z3950_PORT
     volumes:
       - ./masterfiles:/masterfiles
-      - ./gems:/gems
```
Comment (cjcolvar) [inline docker-compose.yml:113]: Do we not need the gems volume anymore?

**Review by cjcolvar — APPROVED** [review]
:+1: Looks fine to me.  I just had a couple questions to double-check things.

File: nginx/Dockerfile (hunk 1 above)
Comment (phuongdh) [reply nginx/Dockerfile:1]: Yes, it's working fine and pretty lightweight. I will revisit it soon though.

File: docker-compose.yml (hunk 2 above)
Comment (phuongdh) [reply docker-compose.yml:113]: No, gems are bundled in the image now.

**Pull request merged**